addons/urdfreader/urdfreader.cc

```diff
@@ -385,8 +475,55 @@ RBDL_ADDON_DLLAPI bool URDFReadFromString(const char *model_xml_string,
     model->gravity.set(0., 0., -9.81);
 
     return true;
+}
+
+RBDL_ADDON_DLLAPI bool PartialURDFReadFromFile(const char *filename,
+                                               Model *model,
+                                               const std::string &root_link,
+                                               const std::vector<std::string> &tip_links,
+                                               bool floating_base, bool verbose)
+{
+  ifstream model_file(filename);
+  if (!model_file) {
+    cerr << "Error opening file '" << filename << "'." << endl;
+    abort();
   }
```

Comment:
I think it would be better not call `abort()` and instead throw an error. I thought I had replaced all occurrences of this pattern when I did the `exception` update years ago. I might have missed the one in the `URDFReadFromFile` function. I can not remember a specific reason that I would have kept it. 🤔

Reply:
Sure, I can do it for you tonight

Reply:
Done